pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/configure-bastion/index.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: layout.pug
+navigationTitle: Configure bastion hosts
+title: Configure bastion hosts
+menuWeight: 60
+excerpt: Configure bastion hosts for access to the DKP cluster
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
+A **bastion host** is a server that provides a defensive barrier to protect access to a private network from an external network, such as the internet.
+
+Bastion hosts are important because, as a cluster administrator, you often want to deploy all of the nodes in a cluster on a private network for security reasons.
```

Comment:
```suggestion
Bastion hosts are important because, as a cluster administrator, you often want to deploy all nodes in a cluster on a private network for security reasons.
```